### PR TITLE
parser: apply feature-ids special comments to auto_random

### PR DESCRIPTION
**Pull request by tangenta**
### What problem does this PR solve?

Problem Summary: For details, see pingcap/parser#777.

### What is changed and how it works?

What's Changed: The result of `show create table` on the `auto_random` table is updated.

How it Works:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test

Side effects

### CODE REVIEW

File: types/parser_driver/special_cmt_ctrl.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+// SpecialCommentVersionPrefix is the prefix of TiDB executable comments.
+const SpecialCommentVersionPrefix = `/*T!`
+
+// BuildSpecialCommentPrefix returns the prefix of `featureID` special comment.
+func BuildSpecialCommentPrefix(featureID string) string {
+	return fmt.Sprintf("%s[%s]", SpecialCommentVersionPrefix, featureID)
+}
+
+const (
```
Comment (AilinKid) [inline types/parser_driver/special_cmt_ctrl.go]: how about adding a comment here describing how we add to a new feature for the followers.

File: types/parser_driver/special_cmt_ctrl.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	FeatureIDAutoRandom = "auto_rand"
+)
+
```
Comment (AilinKid) [inline types/parser_driver/special_cmt_ctrl.go]: how about defining a type
```
type FutureIDType string
const (
      FeatureIDAutoRandom FutureIDType = "auto_rand"
)
```

**Comment by reafans** [comment]
LGTM

**Pull request merged**